index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="Style.css">
+</head>
+<body>
+    
+            <header class="navbar">
+                <nav> 
+                <div> 
+                <a class="logo" >Abdullah</a>
+         
+            <span>
+            <a class="union" href="#" ><img src="Union.png"></a> 
+            </span>
+        </nav>           
+            </div>
+        </header>
```

Comment:
- [ ] As suggested by the previous reviewer, Kindly align the content of your header properly and also add the styling to match the Figma design template you chose. I will recommend you use flexbox to properly align the contents of the header. read about flexbox [here](https://www.w3schools.com/css/css3_flexbox.asp)

I also noticed a background on the header which is not present on the design. KIndly remove it 👍🏽 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/212172372-ee76a3cd-a623-4d8c-9185-4cc3221d3ed2.png) | ![image](https://user-images.githubusercontent.com/63473584/212172885-905104dc-82e6-43c9-bb8f-30e1bb899f04.png)  |